src/main/java/racing/CarRace.java

```diff
@@ -1,10 +1,16 @@
 package racing;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class CarRace {
     private final List<Car> cars;
+    private static final CarMovingStrategy DEFAULT_STRATEGY = new RandomCarMovingStrategy();
```

Comment:
자바 클래스 작성 순서에 따라 상수를 인스턴스 변수보다 위에 배치해보면 어떨까요?
추가로 상수와 인스턴스 변수의 구분을 위해 개행을 추가해보면 좋을 것 같아요.
https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html
```suggestion
    private static final CarMovingStrategy DEFAULT_STRATEGY = new RandomCarMovingStrategy();
    
    private final List<Car> cars;
```